Kickstarter-iOS/Views/Controllers/SheetOverlayContainerViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+import Library
+import Prelude
+
+/* SheetOverlayContainerViewController intended to be used as a container for another view controller
```

Comment:
Thanks for adding the documentation, I think the format we've used elsewhere like in `SharedFunctions.swift` is:

```swift
/**
 SheetOverlayContainerViewController intended to be used as a container for another view controller
 that you would like to render as a "sheet" or "card" that partially covers the content beneath it.
*/
```

Reply:
Oh ok. Will update 👍